From f79d11a2f56e941ed896264dd59f5eed556136f0 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 14 Jul 2020 13:20:48 +0300
Subject: [PATCH] [SW-15841] goya2: remove init for pci

The virtualized PCI events are working fine since Gaudi SoC and there is
no need to handle those events in S/W (although we can).
Remove the PCI module Goya2 specific init and let the H/W handle the
virtualized PCI events.

Change-Id: I3071f8d74cf64f43a82b3de2c97280e37ae90934
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya2/common.c b/goya2/common.c
index 5fa3f6e..3a12922 100644
--- a/goya2/common.c
+++ b/goya2/common.c
@@ -294,12 +294,6 @@
 	spi_copy((void *)loadaddr, offset, size, name);
 }
 
-void pci_dev_specific_setup(void)
-{
-	/* to be handled by SW - enable */
-	regmap_writel(0, mmPCIE_AUX_FLR_CTRL);
-}
-
 u32 hl_pll_timestamp_base_get(void)
 {
 	return mmCPU_TIMESTAMP_CNTCR;
